aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js

```diff
@@ -9,10 +9,16 @@ import { Box, Text } from 'grommet';
 export const NameValueListFormLabel = ({ data, name }) => {
   const theme = useContext(ThemeContext);
   return (
-    <Box margin={{ top: 'small' }} >
+    <Box margin={{ top: 'small' }}>
```

Comment:
Is this file being touched just because of prettier formatting? Seems unrelated to this work.

Reply:
yes it was flagged on my eslint when I ran the `commit`